deps/llvm.mk

```diff
@@ -135,6 +135,14 @@ LLVM_CMAKE += -DLLDB_DISABLE_PYTHON=ON
 endif # LLDB_DISABLE_PYTHON
 endif # BUILD_LLDB
 
+# Part of the FreeBSD libgcc_s kludge
+ifeq ($(OS),FreeBSD)
+ifneq ($(GCCPATH),)
+LLVM_CMAKE += -DCMAKE_INSTALL_RPATH="\$$ORIGIN:$(GCCPATH)"
```

Comment:
All CMake dependencies should have the RPATH include `$ORIGIN`, and indeed we're doing it explicitly and separately for libgit2, libssh2, and mbedtls, so perhaps we should just define it centrally in `CMAKE_COMMON`?